modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -806,6 +873,7 @@ def get_indices(cls, axis, partitions, index_func=None):
         return new_idx[0].append(new_idx[1:]) if len(new_idx) else new_idx
 
     @classmethod
+    @progress_bar_wrapper
     def _apply_func_to_list_of_partitions_broadcast(
```

Comment:
This function doesn't have a progress bar wrapper in the original code - is there a reason why we've added one here?

Reply:
It's a mistake. Good catch!